Tasks/Gradle/gradlesonar.ts

```diff
@@ -27,8 +27,8 @@ export function applyEnabledSonarQubeArguments(gradleRun: trm.ToolRunner):trm.To
     var initScriptPath:string = path.join(__dirname, 'sonar.gradle');
 
     // Specify that the build should run the init script
-    gradleRun.arg(['-I', initScriptPath]);
     gradleRun.arg(['sonarqube']);
+    gradleRun.arg(['-I', initScriptPath]);
```

Comment:
Small unrelated change - I'm adding the "sonarqube" argument before the -I arg so that all the task arguments are grouped